src/core/qgismobileapp.cpp

```diff
@@ -319,6 +319,20 @@ void QgisMobileapp::loadProjectQuirks()
     mLayerTreeCanvasBridge->setAutoSetupOnFirstLayer( true );
 }
 
+void QgisMobileapp::removeRecentProject( const QString &path )
+{
+  QList<QPair<QString, QString>> projects = recentProjects();
+  for ( int idx = 0; idx < projects.count() && idx < 5; idx++ )
```

Comment:
why only < 5 ?

Reply:
Good point -- the max 5 isn't needed here, will update.